dolphinscheduler-api/src/main/java/org/apache/dolphinscheduler/api/service/ProcessDefinitionService.java

```diff
@@ -521,4 +521,15 @@ void saveOtherRelation(User loginUser, ProcessDefinition processDefinition, Map<
      * @return variables data
      */
     Map<String, Object> viewVariables(User loginUser, long projectCode, long code);
+
+    /**
+     * Batch release process definitions by code states.
+     *
+     * @param loginUser    Login user.
+     * @param projectCode  Project code.
+     * @param codeStates   Code states in JSON format.
+     * @return Result of the batch release process definitions.
+     */
+    Map<String, Object> batchReleaseProcessDefinitions(User loginUser, long projectCode, String codeStates);
```

Comment:
do you combine release or unreleased in one single interface? how about add two separate interface `batch release` and `batch unreleased`? which will keep our code more easier to read

Reply:
Hello, thank you very much for your help. I initially wanted to use two methods to express it, until I saw the implementation method below.
https://github.com/apache/dolphinscheduler/blob/dev/dolphinscheduler-api/src/main/java/org/apache/dolphinscheduler/api/controller/ProcessDefinitionController.java#L384
I feel that this method will be more general. Just pass in the corresponding process definition code and corresponding state, and you can process it in bulk and universally. Do you think so?

Reply:
If you want to combine them, maybe use enum class to replace `codeStates` is better, which will also improve code readability.

Reply:
Thank you very much for the suggestion. Perhaps what you said is correct, but the data I sent from the front-end is a jsonstring, with the key being the process definition code and the value being the corresponding state. I feel that using enumeration is not very easy to handle.

Reply:
Can you give me a little hint? I'll do my best. emo.